mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -65,6 +67,77 @@ class Completions(
     (values, result)
   end completions
 
+  private def toCompletionValues(
+      completion: Completion
+  ): List[CompletionValue.Compiler] =
+    val shouldAddSnippet = path match
+      // if the parent is an apply then we already have ()
+      case (fun) :: (appl: GenericApply) :: _ if appl.fun == fun =>
+        false
+      case (_: Import) :: _ => false
+      case _ :: (_: Import) :: _ => false
+      case _ => true
+
+    completion.symbols.flatMap(
+      completionsWithSuffix(_, completion.label, shouldAddSnippet)
+    )
```

Comment:
Maybe put the check on `shouldAddSnippet` here and avoid passing this param to other suffix related functions?
```scala
if shouldAddSnippet && config.isCompletionSnippetsEnabled then
     completion.symbols.flatMap(completionsWithSuffix(_, completion.label))
else
    completion.symbols.flatMap(/*create completionValue*/)
   
```

Reply:
I extracted it anyway and it's also now used for Workspace completions